Doc/whatsnew/3.7.rst

```diff
@@ -1145,9 +1145,10 @@ Changes in Python behavior
   (Contributed by Serhiy Storchaka in :issue:`32012` and :issue:`32023`.)
 
 * When using the ``-m`` switch, the starting directory is now added to sys.path,
-  rather than the current working directory. Any programs that are found to be
-  relying on the previous behaviour will need to be updated to manipulate
-  :data:`sys.path` appropriately.
+  rather than the current working directory. Any programs that are checking for
+  the empty string in :data:`sys.path`, or otherwise relying on the previous
+  behaviour, will need to be updated accordingly (e.g. by checking for
+  ``os.getcwd()`` in addition to checking for the empty string).
```

Comment:
os.getcwd() or the starting directory?

Reply:
Depends on why you're looking for the empty string (in the case of `pydoc` for example, it knows it hasn't changed the working directory, so the starting directory and the current directory are the same thing).

Reply:
By "the current working directory" do you mean the empty string, not os.getcwd()?

The wording here looks slightly confusing.

Reply:
Yeah, I tried to make it clearer in the last set of updates, but it's hard to concisely explain the difference between "current working directory at application startup" and "current working directory at time of import resolution".